Content.Server/_Goobstation/Heretic/Ritual/CustomBehavior.Sacrifice.cs

```diff
@@ -118,10 +187,126 @@ public override void Finalize(RitualData args)
                 && isCommand)
                     crewHeadObjComp.Sacrificed += 1;
             }
+
         }
 
         // reset it because it refuses to work otherwise.
         uids = new();
         args.EntityManager.EventBus.RaiseLocalEvent(args.Performer, new EventHereticUpdateTargets());
     }
+
+    //# funkystation
+    //# sacrifice function to safely teleport them away -space
+    private void SafeSacrifice(TransformComponent transform, RitualData args, EntityUid uid)
+    {
+
+        //drop all items -space
+
+
+        if (args.EntityManager.TryGetComponent<InventoryComponent>(uid, out var comp2))
+        {
+            var transformSystem = _system.EntityManager.System<TransformSystem>();
+            var inventorySystem = _system.EntityManager.System<InventorySystem>();
+            var sharedPopupSystem = _system.EntityManager.System<SharedPopupSystem>();
+
+            foreach (var item in inventorySystem.GetHandOrInventoryEntities(uid))
+            {
+                transformSystem.DropNextTo(item, uid);
+            }
+        }
+
+
+
+
+        // teleports them away -space
+        var mapPos = _xform.GetWorldPosition(transform);
+        var radius = 30;
+        var gridBounds = new Box2(mapPos - new Vector2(radius, radius), mapPos + new Vector2(radius, radius));
+
+        var mobs = new HashSet<Entity<MobStateComponent>>();
+        _lookup.GetEntitiesInRange(transform.Coordinates, .0000001f, mobs, flags: LookupFlags.Uncontained);
+        if (args.EntityManager.TryGetComponent<MobStateComponent>(uid, out var mobstate))
```

Comment:
Are you certain this is needed?

Reply:
oh my god how tf did i forget to delete that